lib/config/config.js

```diff
@@ -159,7 +159,15 @@ module.exports = function (fs, path, url, convict) {
         'restmail.net',
         'mockmyid.com'
       ]
-    }
+    },
+    blocklist: {
+      listPath: {
+        doc: 'Location of the csv file containing list of ip addresses to block',
```

Comment:
does this need to state that this relative to the fxa-customs-server dir? (because it uses ` csvPath = path.join(__dirname, csvPath)` ?